### PR TITLE
Timezones Fixed for 1.9.6

### PR DESCRIPTION
**Pull request by ImAFrogOwO**

### CODE REVIEW

File: Timezones/Timezones.plugin.js [hunk 1]
```diff
@@ -132,37 +132,41 @@ module.exports = (() => {
...
                           ContextMenu.patch("user-context", this.userContextPatch);
 
-                          Patcher.after(ProfileBanner, "Z", (_, [props], ret) => {
+                          Patcher.after(ProfileBanner, "default", (_, [props], ret) => {
                               const originalRet = { ...ret };
-
-                              if (!this.hasTimezone(props.user.id)) return;
-
-                              ret.type = "div";
+                              console.log("Banner:", _,props,ret)
```
Comment (TheCommieAxolotl) [inline Timezones/Timezones.plugin.js:149]: Remove this

File: Timezones/Timezones.plugin.js [hunk 2]
```diff
@@ -132,37 +132,41 @@ module.exports = (() => {
...
+                  const Markdown = Webpack.getModule((m) => m.default?.rules && m.default?.defaultProps?.parser).default
 
                   return class Timezones extends Plugin {
                       async onStart() {
                           injectCSS("Timezones-Styles", Styles);
 
-                          const ProfileBanner = Webpack.getModule((m) => m.Z?.toString().includes("e.displayProfile") && m.Z?.toString().includes("e.user"));
-                          const MessageHeader = Webpack.getModule((m) => m.Z?.toString().includes("userOverride") && m.Z?.toString().includes("withMentionPrefix"));
-                          const Tooltip = BdApi.Components.Tooltip;
+                          const ProfileBanner = Webpack.getModule(x=>x.default && x.default.toString().includes("y.GifAutoPlay")) //Webpack.getModule(x=>x.default && x.default.toString().includes("c.AvatarDecorationBorderSizes"))
```
Comment (TheCommieAxolotl) [inline Timezones/Timezones.plugin.js:141]: Why the comment?

**Review by TheCommieAxolotl — CHANGES_REQUESTED** [review]

File: Timezones/Timezones.plugin.js (hunk 2 above)
Comment (ImAFrogOwO) [reply Timezones/Timezones.plugin.js:141]: I added the comment to make sure to keep it stored just incase "y.GifAutoPlay" wasnt right

**Comment by ImAFrogOwO** [comment]
Should be fixed now.

**Pull request merged**